tests/batching_test.py

```diff
@@ -539,6 +539,122 @@ def testLaxLinalgCholesky(self):
     expected = onp.linalg.cholesky(b)
     self.assertAllClose(ans, expected, check_dtypes=False)
 
+  @parameterized.named_parameters(
+      {"testcase_name": "_shape={}_axis={}_idxs={}_dnums={}_slice_sizes={}".format(
+          jtu.format_shape_dtype_string(shape, dtype), axis, idxs, dnums,
+          slice_sizes),
+       "axis": axis, "shape": shape, "dtype": dtype, "idxs": idxs, "dnums": dnums,
+       "slice_sizes": slice_sizes, "rng": rng, "rng_idx": rng_idx}
+      for dtype in [onp.float32, onp.int32]
+      for axis, shape, idxs, dnums, slice_sizes in [
+          (0, (3, 5), onp.array([0, 2]), lax.GatherDimensionNumbers(
+            offset_dims=(), collapsed_slice_dims=(0,), start_index_map=(0,),
+            index_vector_dim=1), (1,)),
```

Comment:
These all test index_vector_dim == 1 with a scalar index. Might be worth adding at least one vector index case too.

Reply:
Makes sense, though iiuc we should really add that to lax_test.py's testGather (and then include a derived test case here, since all these cases are derived from those). I'll look to add one of those.

Reply:
Done!